.changes/add-total-transferred-field.md

```diff
@@ -0,0 +1,6 @@
+---
+"upload": "minor"
+"upload-js": "minor"
+---
+
+Added a new field `total_transferred` (or `progressTotal` in JS) to track the total amount of data transferred during the upload/download process.
```

Comment:
```suggestion
Added a new field `progressTotal` to track the total amount of data transferred during the upload/download process.
```
The rust side is not exposed outside of the crate so our users don't have to know about this here.